apps/web/server/lib/[user]/[type]/getServerSideProps.ts

```diff
@@ -219,6 +219,12 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
     } as const;
   }
 
+  const allowSEOIndexing = org
```

Comment:
Followup: Abstract it out into something like `packages/lib/isSeoIndexableForUser` and reuse it at all places where we derive user's seoIndexing setting from org and user.